resources/master_messages.json

```diff
@@ -357,6 +357,16 @@
       }
     }
   },
+  "error_shadowsocks_unsupported_cipher": {
+    "message": "Shadowsocks Unsupported cipher",
+    "description": "The server you added uses cipher $cipher$, but Outline only supports AEAD ciphers. Please contact your server admin.",
```

Comment:
Maybe something like "The server you added uses cipher $cipher$, which is not safe. Please contact your server admin"

I think it's important to say it's not safe.

Reply:
That sounds good. I tweaked the verbiage a bit.